js/ui/toolbar.js

```diff
@@ -323,7 +323,14 @@ const Toolbar = ToolbarBase.inherit({
 
     _changeMenuOption: function(name, value) {
         this._menuStrategy.widgetOption(name, value);
-    }
+    },
+
+    _setDeprecatedOptions() {
+        this.callBase();
+        extend(this._deprecatedOptions, {
+            'height': { since: '20.1', message: 'This option is deprecated, because the functionality controlled by it was not supposed to belong to the Toolbar widget' }
```

Comment:
```suggestion
            'height': { since: '20.1', message: 'Functionality associated with this option is not intended for the Toolbar widget.' }
```